unity/UnityEmbedHost.Tests/BaseEmbeddingApiTests.cs

```diff
@@ -616,6 +616,18 @@ static void AssertMultiDimensionalArraysAreEquivalent(Array expected, Array actu
         Assert.That(actualFlat, Is.EquivalentTo(expectedFlat));
     }
 
+    [TestCase(typeof(Classification),              true)]
+    [TestCase(typeof(Classification*),            false)]
+    [TestCase(typeof(Classification[]),           false)]
+    [TestCase(typeof(Animal),                     false)]
+    [TestCase(typeof(IAnimal),                    false)]
+    [TestCase(typeof(GenericAnimal<int, string>), false)]
```

Comment:
Nit, one more test should be an enum inside as generic as this is a common corner case causing errors.

struct Foo<T>
{
    public enum Bar { One, Two };
}